sample_shavar_list_creation.ini

```diff
@@ -76,3 +76,23 @@ output=mozplugin2-block-digest256
 [entity-whitelist]
 entity_url=https://s3.amazonaws.com/lists.disconnect.me/exampleservices.json
 output=mozpub-trackwhite-digest256
+
+[fastblock1]
+blocklist=https://raw.githubusercontent.com/mozilla-services/shavar-experiments/master/fastblock1.json
+output=fastblock1-track-digest256
+
+[fastblock1-whitelist]
+entity_url=https://raw.githubusercontent.com/mozilla-services/shavar-experiments/master/fastblock1-whitelist.json
+output=fastblock1white-track-digest256
```

Comment:
The list format here requires that the middle portion be `-trackwhite-`, so this would be `fastblock1-trackwhite-digest256`.